packages/api/src/schema/api-schema.yaml

```
@@ -2112,60 +2198,22 @@ components:
             - public
             - jwt
             - webhook
+          example: webhook
         webhookId:
           type: string
           description: ID of the webhook to use for playback policy
+          example: 1bde4o2i6xycudoy
         webhookContext:
           type: object
           description: User-defined webhook context
           additionalProperties: true
+          example: {}
```

Maybe add something in the example here for easier understanding. WDYT?
```suggestion
          example: {"streamerId":"my-custom-id"}
```